plugins/logger/CMakeLists.txt

```diff
@@ -149,6 +149,11 @@ function(generatePluginsLoggerKafkaproducer)
     add_osquery_library(plugins_logger_kafkaproducer EXCLUDE_FROM_ALL
       kafka_producer.cpp
     )
+  if(DEFINED PLATFORM_WINDOWS)
```

Comment:
The if blocks checking the platform can be removed

Reply:
block removed